releases/release-1.31/release-notes/maps/pr-120699-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 120699
```

Comment:
Let's delete this map file- it has `"do_not_publish": true,` so it should be correctly rendered without a release note. With the map it's rendering:
```
 NONE ([#120699](https://github.com/kubernetes/kubernetes/pull/120699), [@liyuerich](https://github.com/liyuerich))
```

This can be done in the final release notes draft as well to not hold this PR up.